### PR TITLE
Utils: Remove unused functions and reduce method scope

### PR DESCRIPTION
**Pull request by danrabbit**

### CODE REVIEW

File: core/Utils/PlaylistsUtils.vala [hunk 1]
```diff
@@ -111,7 +111,7 @@ namespace Music.PlaylistsUtils {
         return rv;
     }
 
-    public static bool parse_paths_from_m3u(string uri, ref Gee.LinkedList<string> locals) {
+    private static bool parse_paths_from_m3u(string uri, ref Gee.LinkedList<string> locals) {
```
Comment (jeremypw) [inline core/Utils/PlaylistsUtils.vala]: Missing space before bracket.

File: core/Utils/PlaylistsUtils.vala [hunk 2]
```diff
@@ -145,7 +145,7 @@ namespace Music.PlaylistsUtils {
         return true;
     }
 
-    public static bool parse_paths_from_pls(string uri, ref Gee.LinkedList<string> locals, ref string title) {
+    private static bool parse_paths_from_pls(string uri, ref Gee.LinkedList<string> locals, ref string title) {
```
Comment (jeremypw) [inline core/Utils/PlaylistsUtils.vala]: Missing space before bracket.

File: core/Utils/PlaylistsUtils.vala [hunk 3]
```diff
@@ -182,7 +182,7 @@ namespace Music.PlaylistsUtils {
         return true;
     }
 
-    public static void parse_index_and_value(string prefix, string line, ref Gee.HashMap<int, string> map) {
+    private static void parse_index_and_value(string prefix, string line, ref Gee.HashMap<int, string> map) {
```
Comment (jeremypw) [inline core/Utils/PlaylistsUtils.vala]: Missing space before bracket.

**Comment by jeremypw** [comment]
Presumably no implication for 3rd party users of Music core library (if any)?  Should not affect 3rd party plugins using proper interface.

**Comment by danrabbit** [comment]
AFAIK there are no 3rd party plugins, only the ones that we ship with Music

**Pull request merged**